package.json

```diff
@@ -8,7 +8,16 @@
     "start": "next start",
     "lint": "next lint",
     "gen-demo": "plop",
-    "format": "prettier --write ."
+    "format": "prettier --write .",
+    "prepare": "husky install"
+  },
+  "lint-staged": {
+    "*.{js,jsx}": [
+      "eslint --fix"
```

Comment:
Just an fyi - if you want to avoid committing code with lint warnings/errors, then you can change the command to 
`eslint --fix --max-warnings 0`.
 
 This sets a warning threshold, and protects warning-level rule violations in your project.

Reply:
I think we can keep it a little loose without a warning threshold for the time being